pkg/js/parse_tests/015-tlsa/foo.com.zone

```diff
@@ -1,2 +1,2 @@
 $TTL 300
-_443._tcp        IN TLSA  3 1 1 MDFiYTQ3MTljODBiNmZlOTExYjA5MWE3YzA1MTI0YjY0ZWVlY2U5NjRlMDljMDU4ZWY4Zjk4MDVkYWNhNTQ2YiAgLQo=
+_443._tcp        IN TLSA  3 1 1 mdfiytq3mtljodbinmzlotexyja5mwe3yza1mti0yjy0zwvly2u5njrlmdljmdu4zwy4zjk4mdvkywnhntq2yiaglqo=
```

Comment:
I'm surprised that this needed to change for the tests to pass.  These tests are agnostic to the providers.  Was this change necessary?

Reply:
You are absolutely right.  I swear that without that change, unit tests failed.  It just proves that I shouldn't swear.  :)
I changed it back to the original.

Reply:
The automated build failed like this:
--- FAIL: TestParsedFiles (0.88s)
18
    --- FAIL: TestParsedFiles/015-tlsa.js (0.02s)
19
        js_test.go:117: 
20
            	Error Trace:	js_test.go:117
21
            	Error:      	Not equal: 
22
            	            	expected: "$TTL 300\n_443._tcp        IN TLSA  3 1 1 MDFiYTQ3MTljODBiNmZlOTExYjA5MWE3YzA1MTI0YjY0ZWVlY2U5NjRlMDljMDU4ZWY4Zjk4MDVkYWNhNTQ2YiAgLQo=\n"
23
            	            	actual  : "$TTL 300\n_443._tcp        IN TLSA  3 1 1 mdfiytq3mtljodbinmzlotexyja5mwe3yza1mti0yjy0zwvly2u5njrlmdljmdu4zwy4zjk4mdvkywnhntq2yiaglqo=\n"

It appeared to pass on my local system because I had run the test successfully before and the test had been "cached".

To clear the cached tests:
 go clean -testcache

After that, the unit test failed on my local machine.

pkg/js/parse_tests/015-tlsa/foo.com.zone once again contains lowercase.  The test passes.